osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -75,7 +75,7 @@ private static double calculateNormalisedHitError(double od, int objectCount, in
         {
             int circle300Count = count300 - (objectCount - circleCount);
             if (circle300Count <= 0)
-                return double.NaN;
+                return 0;
```

Comment:
Changing this has an impact on output.
https://github.com/Syriiin/osu/blob/9f1f39f52548c43a425f581e19920ef07b9306e4/osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs#L122
With this change, it would result in `560` instead of `0`.
It could be refactored, but I don't think that's relevant to this bug, so let's revert this for now.

Reply:
Reverted in fa30ebb83afa8fce31c447c878e54cba23610cb6

Reply:
https://osu.ppy.sh/beatmapsets/1713744#osu/3544157
check this map, except SS or this will return double.NaN to `normalisedHitError` and all skill will be 0 pp

![image](https://github.com/Syriiin/osu/assets/34775378/b3a2a606-eee4-42e3-bf22-32d0333783d4)